server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -344,6 +346,19 @@ export class AssetController {
     return await this.assetService.checkExistingAssets(authUser, checkExistingAssetsDto);
   }
 
+  /**
+   * Checks if assets exist by checksums
+   */
+  @Authenticated()
+  @Post('/bulk-upload-check')
+  @HttpCode(207)
```

Comment:
Why response code `207`?

Reply:
I think the original idea was to communicate that the response would containe multiple response codes, one for each item. However, we aren't returning multiple _http response_ codes, just our own custom enums, so I think a 200 would probably 
be more appropriate.

Reply:
Fixed